examples/basic/basic.js

```diff
@@ -36,4 +34,4 @@ horde.mogwai(horde.mogwais.fps());
 horde.mogwai(horde.mogwais.gizmo().maxErrors(2));
 
 // Unleash the horde
-horde.unleash({ nb: 1000, delay: 1 });
+horde.unleash({ nb: 200, delay: 10 });
```

Comment:
Why do you change this value?

Reply:
1ms delay is too short and 1000 gremlins is a lot.